src/components/inline/InputText/InputText.js

```diff
@@ -1,8 +1,30 @@
-import PropTypes from "prop-types";
-import React from "react";
-
-import s from "./InputText.module.css";
-
+"use strict";
```

Comment:
I think we definitely should not be committing the JavaScript file being generated from TypeScript to git. The generated .js code should just be considered a build artifact, not a source file.

Reply:
I guess until all the javascript files turns into typescript. Careful staging of files must be stressed. Then we can put a regular expression in the .gitignore right?

Reply:
It still feels a bit too error prone to me to have to carefully add only the .tsx files and not the generated .js files, since the .js files end up being generated in the same directory as the other files, so I'm almost certain all of us will accidentally commit them as we're working on stuff.

I'll try to do some research on different ways people do this for TypeScript codebases, since the way this plugin works right now seems pretty weird to me.

Reply:
Off the top of my head I think there are two ways we can resolve this:
1. We can use the `outDir` config option to output all compiled files into one directory, which we can then add to gitignore. Check out [this stackoverflow conversation](https://stackoverflow.com/questions/24454371/how-can-i-get-the-typescript-compiler-to-output-the-compiled-js-to-a-different-d) about it.
2. We can use the `noEmit` config option, which stops it from outputting compiled .js files in the first place. I think this would work fine for us, since we're not relying on Typescript itself for final compilation anyway (Gatsby handles that, with webpack/babel under the hood).

But I don't think I see a `tsconfig.json` file in this PR? We're going to need one to configure things like this for Typescript. Adding one is really easy -- Gatsby even seems to provide [their own example file](https://github.com/gatsbyjs/gatsby/blob/master/examples/using-typescript/tsconfig.json), so copy/pasting that would probably make a good starting point. And for understanding the different options, I like [this resource](https://medium.com/javascript-in-plain-english/typescript-configuration-options-tsconfig-json-561d4a2ad4b).

Reply:
Hah -- sorry to overlap you on this one, Richard!

Reply:
Hah, no problem, looks like we posted it at right about the same time! I think you have more experience than I do on this, @jessypeck, but I think these sound like good paths to try out.

Note that I think we may still want the CSS modules files to create `.d.ts` files, since they'll be needed in order for TypeScript to understand the types of those CSS files when import, but if there's a way for the tools to understand the CSS files as TypeScript files without needing to generate the files, that'd be great.

Reply:
Yeah, I think the .d.ts files can get too messy with our code. I'm researching some alternative for it.